Tools/PowershellModule/doc/PowerShell/Get-WinGetManifest.md

```diff
@@ -46,35 +41,35 @@ The following Azure Modules are used by this script:
 Get-WinGetManifest -Path "C:\AppManifests\Microsoft.PowerToys"
 ```
 
-Returns an array of all Manifest objects based on the files found within the specified Path.
+Returns an array of the Application Manifest objects based on the files (*.yaml or *.json) found within the specified Path.
 
 ### EXAMPLE 2
 ```
 Get-WinGetManifest -Path "C:\AppManifests\Microsoft.PowerToys\Microsoft.PowerToys.json"
 ```
 
-Returns a Manifest object (*.json) of the specified JSON file.
+Returns an Application Manifest object (*.json) of the specified JSON file.
 
 ### EXAMPLE 3
 ```
 Get-WinGetManifest -FunctionName "PrivateSource" -ManifestIdentifier "Windows.PowerToys"
```

Comment:
Private reference

Reply:
no..? That doesn't show in my content.

Reply:
Your content is either old or you arent pushing the updates.

Reply:
fixed